site/docs/v1/tech/client-libraries/netcore.adoc

```diff
@@ -37,11 +37,101 @@ include::docs/v1/tech/client-libraries/_static-patch-note.adoc[]
 :client_library_language!:
 
 
+== FusionAuth Templates for .NET
+
+FusionAuth has a collection of .NET templates available as a NuGet package. Quickly create new secured .NET applications with FusionAuth templates using the .NET CLI or Visual Studio. The new applications are pre-configured to use FusionAuth for authentication and authorization. Only standard .NET security libraries are used in the templates.
+
+Currently, the following .NET templates are implemented:
+
+* FusionAuth Blazor Server Application.
+* FusionAuth MVC Application.
+* FusionAuth Web API Application.
+
+
+=== Requirements
+
+* A FusionAuth installation. You can link:/docs/v1/tech/installation-guide/[install FusionAuth locally, or sign up for a hosted account].
+* .NET Core 7.0 SDK or higher.
+* Visual Studio 2022 for Mac, version 17.6 and above (optional).
+* Visual Studio 2022 for Windows, version 17.6 and above (optional).
+
+
+=== Installing the Templates
+
+FusionAuth .NET templates are available on link:https://www.nuget.org/packages/FusionAuth.Templates/[NuGet]. You can install them by running the following command in your terminal.
+
+```bash
+dotnet new install FusionAuth.Templates::1.0.0
+```
+
+When installed successfully, the templates will be available in the .NET CLI and Visual Studio. The installation is the same for both Windows and macOS.
+
+=== Using the Templates
+
+To help you set up a valid application in FusionAuth, we have created a FusionAuth link:/docs/v1/tech/installation-guide/kickstart[Kickstart], available on https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/Kickstart/kickstart/kickstart.json[GitHub]. Follow the instructions in the https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/Kickstart/README.md[README.md] file to set up FusionAuth for your project. If you don't want to use the Kickstart file, you can set up your application manually in FusionAuth, using the values referenced in the https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/Kickstart/kickstart/kickstart.json[kickstart.json] file.
+
+Note that the Kickstart is designed to be used when starting up FusionAuth for the first time using `docker compose up`.
```

Comment:
Please take a look at the "Run FusionAuth via Docker" section on how to explain getting the instance running.
[Run FusionAuth via Docker](https://fusionauth.io/docs/quickstarts/quickstart-ruby-rails-web)

Reply:
@mark-robustelli we had covered the instructions in detail in the README.md here https://github.com/FusionAuth/fusionauth-example-client-libraries/pull/8/files which is part of the PR that is open https://github.com/FusionAuth/fusionauth-example-client-libraries/blob/d134775a3b1cdca558916f7785e3d0f8275a980e/dotnet/Kickstart/README.md